samanthas_telegram_bot/application_start/bot_data_loader.py

```diff
@@ -37,16 +25,14 @@ class BotDataLoader:
     """Class for loading bot data from backend or external files.
 
     Should be called in Application.post_init() to refresh data that were loaded from persistence.
-
-    Note:
-        ``BackendClient`` cannot be used here to get data because all its public methods
-        require access to an update that is not yet available when bot is loading.
     """
 
     @classmethod
-    def load(cls, bot_data: BotData) -> None:
+    def loads(cls, bot_data: BotData) -> None:
```

Comment:
Usually in python `.loads` is "`.load` from string". Why change here?

Reply:
I just wanted to make sure it was a verb and didn't realize I actually put `s` for "string" :) I'll change it back